rollup.utils.js

```diff
@@ -223,17 +206,15 @@ function magicExportsPlugin({ packageName, version }) {
           )
           .join("\n");
 
-        cjsContents +=
-          `var ${moduleName} = require('${packageName}');\n` +
-          deprecationFunctions;
-        cjsContents += magicExports.values
-          .map(
-            (symbol) =>
-              `/** @deprecated Import \`${symbol}\` from \`${packageName}\` instead. */\n` +
-              `const ${symbol} = warn(${moduleName}.${symbol}, getDeprecatedMessage('${symbol}', '${packageName}'));\n` +
-              `exports.${symbol} = ${symbol};\n`
-          )
-          .join("\n");
+        let cjsModule = camelCase(packageName.slice("@remix-run/".length));
+        cjsContents += `var ${cjsModule} = require('${packageName}');\n`;
+        for (let symbol of magicExports.values) {
+          cjsContents +=
+            `Object.defineProperty(exports, '${symbol}', {\n` +
+            "  enumerable: true,\n" +
+            `  get: function () { return ${cjsModule}.${symbol}; }\n` +
+            "});\n";
+        }
```

Comment:
I think we can keep the JSdoc deprecation here as well to have an extra IDE warning?

```suggestion
        cjsContents += `var ${moduleName} = require('${packageName}');\n`;
        cjsContents += magicExports.values
          .map(
            (symbol) =>
              `/** @deprecated Import \`${symbol}\` from \`${packageName}\` instead. */\n` +
              `exports.${symbol} = ${symbol};\n`
          )
          .join("\n");
```